nss-day-cohort-41/tabloid-fullstack-elvis_spotters
# Aj refactor routes

# Description
ReFactored Routes for easier additions of future routes; Notes in code.
Fixes N/A
## Type of change
# Testing Instructions
-Application should run as before.
-Varify you are having no additional issues with routing.
-Check Both logged in and logged out
-Unknown routes should reroute to a 404
# Checklist:
